docs-source/migrating-to-v4.md

```diff
@@ -0,0 +1,132 @@
+# Migration Guide
+
+## Migrating from v3 to v4
+
+Version 4 of both the Authentication API SDK as well as the Management API SDK are major breaking changes. This document will discuss the reason for these changes, as well as instructions on how to migrate to the new version.
+
+### OIDC Conformance
+
+The main reason for the breaking changes is related to the OIDC-conformance which was added to the Authentication API. Because of this, the behaviour of some of the existing Authentication API endpoints have changed, and other endpoints are being deprecated.
```

Comment:
```suggestion
The main reason for the breaking changes is related to improved OIDC compliance which was added to the Authentication API. Because of this, the behaviour of some of the existing Authentication API endpoints have changed, and other endpoints are being deprecated.
```